bridge-history-api/internal/logic/l1_event_parser.go

```diff
@@ -168,6 +168,14 @@ func (e *L1EventParser) ParseL1SingleCrossChainEventLogs(ctx context.Context, lo
 			lastMessage.L2TokenAddress = event.L2Token.String()
 			lastMessage.TokenIDs = utils.ConvertBigIntArrayToString(event.TokenIDs)
 			lastMessage.TokenAmounts = utils.ConvertBigIntArrayToString(event.TokenAmounts)
+		case backendabi.L1DepositWrappedTokenSig:
+			event := backendabi.WrappedTokenMessageEvent{}
+			if err := utils.UnpackLog(backendabi.L1WrappedTokenGatewayABI, &event, "DepositWrappedToken", vlog); err != nil {
+				log.Error("Failed to unpack DepositWrappedToken event", "err", err)
+				return nil, nil, err
+			}
+			lastMessage := l1DepositMessages[len(l1DepositMessages)-1]
+			lastMessage.Sender = event.From.String()
```

Comment:
also update `to` fields.
token type (eth) and amount are optional, for readability considerations suggesting add them, but it's up to you.

Reply:
Yes, here I'm thinking that, because we take is as a normal erc20 token, so before the `DepositWrappedToken` event,  there is a `L1DepositERC20` event emit, we have already set `to`, `amount` and `tokentype` there.

Reply:
makes sense. resolved.